src/OSPSuite.SimModelNative/OSPSuite.SimModelNative.vcxproj

```diff
@@ -77,11 +79,12 @@
       <RuntimeLibrary>MultiThreadedDebugDLL</RuntimeLibrary>
       <PrecompiledHeader>NotUsing</PrecompiledHeader>
       <WarningLevel>Level3</WarningLevel>
-      <DebugInformationFormat>ProgramDatabase</DebugInformationFormat>
+      <DebugInformationFormat>OldStyle</DebugInformationFormat>
```

Comment:
??

Reply:
Required to get full info (line numbers/function names) for the deleaker tool. I will write it in the Wiki later (s. below)
It's actually switch to [/Z7](https://docs.microsoft.com/en-us/cpp/build/reference/z7-zi-zi-debug-information-format?view=vs-2019)